### PR TITLE
Add tests for importing beacon data

### PR DESCRIPTION
**Pull request by MalinAhlberg**
### Describe the pull request:

- [X] New feature

### Pull request long description:

This is a start on beacon oriented tests for the import scripts. They will probably need more work and maybe some restructuring, but I add them as a PR already now in an attempt to make things less confusing when coming back from vacation.

Two new things are tested: the dataset `variantcount` and `callcount`, and the parsing of manta files.

### Changes made:

- Adds test for adding data set counts (`--count_calls`)
    - updates `travis_script.sh` to count these calls for `dataset1_1` and `dataset2_1`
    - add expected counts to `reference.psql`

- Adds test for importing a manta file (ie mates, breakends)
    - test data (`manta.vcf`, slightly modified Swegen data)
    - reference counts (`mates_reference.txt`)
   - update to `travis_script.sh`, `Test 6`, to make an import and compare the result.

### CODE REVIEW

File: scripts/importer/tests/data/reference.psql [hunk 1]
```diff
@@ -1,4 +1,7 @@
-COPY beacon.beacon_dataset_counts_table (datasetid, dataset, callcount, variantcount) FROM stdin;
+COPY beacon.beacon_dataset_counts_table (datasetid, callcount, variantcount) FROM stdin;
```
Comment (talavis) [inline scripts/importer/tests/data/reference.psql]: I will probably rewrite this file from scratch before I'm ready for merging this, so this part will probably have to be redone.

File: test/travis_script.sh [hunk 2]
```diff
@@ -160,4 +162,19 @@ if [ -f .coverage ]; then
...
 fi
 
+echo '>>> Test 6. Reading manta file'
+
+
+./scripts/manage.sh import --add_raw_data \
+                   --dataset "Dataset 1" \
+                   --version "Version 1" \
+                   --add_mates \
+                   --assembly_id "GRCh37p13" \
```
Comment (talavis) [inline test/travis_script.sh]: This value is connected to `var_call_ref` which is no longer used. `--assembly_id` is used when loading reference sets instead.

File: scripts/importer/tests/data/reference.psql [hunk 3]
```diff
@@ -1,4 +1,7 @@
...
+GRCh37p13:Dataset 1:2001-01-01 00:00:00	18	18
+GRCh37p13:Dataset 2:2001-01-01 00:00:00	12	15
+
```
Comment (talavis) [inline scripts/importer/tests/data/reference.psql]: Newlines are kept out of the reference file

File: scripts/importer/tests/data/reference.psql [hunk 4]
```diff
@@ -1,4 +1,7 @@
-COPY beacon.beacon_dataset_counts_table (datasetid, dataset, callcount, variantcount) FROM stdin;
+COPY beacon.beacon_dataset_counts_table (datasetid, callcount, variantcount) FROM stdin;
+GRCh37p13:Dataset 1:2001-01-01 00:00:00	18	18
+GRCh37p13:Dataset 2:2001-01-01 00:00:00	12	15
```
Comment (talavis) [inline scripts/importer/tests/data/reference.psql]: Running this gives me:
```
686c687
< COPY beacon.beacon_dataset_counts_table (datasetid, dataset, callcount, variantcount) FROM stdin;
---
> COPY beacon.beacon_dataset_counts_table (datasetid, callcount, variantcount) FROM stdin;
743,744c744,745
< GRCh13:Dataset 1:Version 1	\N	18	18
< GRCh13:Dataset 2:Version 1	\N	12	17
---
> GRCh13:Dataset 1:2001-01-01 00:00:00	18	18
> GRCh13:Dataset 2:2001-01-01 00:00:00	12	15
```

File: test/travis_script.sh [hunk 5]
```diff
@@ -160,4 +162,19 @@ if [ -f .coverage ]; then
     coverage report
 fi
 
+echo '>>> Test 6. Reading manta file'
+
+
```
Comment (talavis) [inline test/travis_script.sh]: Make sure to get a coverage report for the import using e.g.
`sed 's/import_4/mate_1/'`
Also make sure the filename (`.coverage_mate_1` in the example above) is added to the `coverage combine` line.

Also: Test 6 should come before Finalizing.

File: scripts/importer/tests/data/reference.psql (hunk 4 above)
Comment (MalinAhlberg) [reply scripts/importer/tests/data/reference.psql]: Yes, looks like I somehow miscalculated the variantcount. Fixed in 28f42d1e065ba774ca4c0da0ea4085f4acdcb4db.

**Pull request merged**